site2/docs/client-libraries-python.md

```diff
@@ -64,7 +66,7 @@ client.close()
 
 ### Consumer example
 
-This creates a consumer with the `my-subscription` subscription on the `my-topic` topic, listen for incoming messages, print the content and ID of messages that arrive, and acknowledge each message to the Pulsar broker:
+The following example creates a consumer with the `my-subscription` subscription on the `my-topic` topic, listens for incoming messages, prints the content and ID of messages that arrive, and acknowledges each message to the Pulsar broker.
```

Comment:
```suggestion
The following example creates a consumer with the `my-subscription` subscription name on the `my-topic` topic, listens for incoming messages, prints the content and ID of messages that arrive, and acknowledges each message to the Pulsar broker.
```

Reply:
BTW, maybe use receives for in coming messages to more reasonable here, because consumer demo is use receive method to consume messages, also we have consumer listeners api there.

Reply:
@codelipenghui Thanks for your advice. Updated. PTAL, thank you.